### DISCUSSION TITLE
Adding Cursor property to UIElement

### DISCUSSION THREAD

**Discussion by HHaoWang** [microsoft/WindowsAppSDK#3910] [Polls]
As mentioned here: #1816 . I can not understand why we can not change an element's cursor easily and even there is no way to change a sealed element's cursor (excepts using reflection, but I really dont want to use it due to performance and aot). And according to this issue's answer, now that we have now a protected cursor property why the library can not make it public? It is a basic function that cannot be further basic. I'm really confused.

**Comment by DarranRowe** [comment]
Does directly using [UIElement.ProtectedCursor](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.uielement.protectedcursor?view=windows-app-sdk-1.4#microsoft-ui-xaml-uielement-protectedcursor) not work for you? This is what the reflection's target is, so you should just use it directly.

**Reply by HHaoWang** [reply]
Now that it is a protected property there are several scenarios it not working(a classic example is border which is a sealed class and you can not create a subclass to do so)

**Comment by benstevens48** [comment]
I agree, this is quite important for me. In addition, I have a requirement to hide the cursor. I would be happy if there was an easy way to set the cursor on the window as in UWP. Not sure exactly how this would interact with the cursor property on the UIElement - may need to be able to specify which takes precedence.

It might also be useful if the cursor property could be set via a style, like in HTML/CSS.
